Sources/AmplifyUtilsNotifications/AUNotificationPermissions.swift

```diff
@@ -11,8 +11,13 @@ import UserNotifications
 #if canImport(WatchKit)
 import WatchKit
 #elseif canImport(AppKit)
+#if targetEnvironment(macCatalyst)
+import UIKit
+typealias Application = UIApplication
+#else
```

Comment:
I think this can be simplified to:
```swift
// ...
#elseif canImport(AppKit) && !targetEnvironment(macCatalyst)
import AppKit
typealias Application = NSApplication
#elseif canImport(UIKit)
import UIKit
typealias Application = UIApplication
// ...
```

Do you think that would help with readability?

Reply:
Or even just reordering so that the UIKit check is first.
```swift
// ...
#elseif canImport(UIKit)
import UIKit
typealias Application = UIApplication
#elseif canImport(AppKit)
import AppKit
typealias Application = NSApplication
// ...
```